docs/docs/contributions/releases/release-process.mdx

```diff
@@ -158,6 +158,26 @@ Once the workflow finishes, look through any failures and determine if there's a
 
 Alternatively, after starting the workflow, post the link to the in-progress run in `#development` in Slack, so that someone can come back to it when it does finish.
 
+## Step 5: Publish a schema in JSON Schema Store
+
+Pants configuration file schema is published at https://www.schemastore.org/. For every release, a new schema needs to generated
+and uploaded by submitting a PR against https://github.com/SchemaStore/schemastore.
+
+To produce `pantsbuild-<version>.json` schema file, run:
+
+```bash
+pants help-all > all-help.json
+pants run build-support/bin/generate_json_schema.py -- --all-help-file=all-help.json
+```
+
+It may be helpful to compare the last schema file with the newly produced to make sure there are no discrepancies
+(e.g. the config values have a sensible type and the help strings are rendered adequately).
+
+Watch out for any configuration parameters that may rely on your local environment as certain default config values
+will be expanded using local runtime environment which is undesirable. The script handles those known config values
+by keeping a list of them, however, it may need to be extended as more options with environment specific default
+values are added.
+
```

Comment:
Could you add a link to an example pull request that you've done? I personally find that sort of "here's a previous example" to be a helpful reference for this sort of task.

Reply:
For sure, added one!